### DISCUSSION TITLE
在更多的候选栏中增加一些下载MOD，材质包，wiki的资源网站快速引导栏目以便于查找资源等相关建议

### DISCUSSION THREAD

**Discussion by Bad0RANG3** [Meloong-Git/PCL#2460] [提问] [answered]
1、在更多的候选栏中增加一些下载MOD，材质包，wiki的资源网站快速引导栏目以便于查找资源
2、在下载的候选框里增加材质包的下载选项，虽然知道可能比较难实现但还是辛苦龙猫了
3、快照版赞助了**23.33元及以上**的隐藏快照版提示哪里可以直接输入解锁码然后直接更新新的PCL2启动器
4、设置里的输入解锁码和更多里的关于与鸣谢里的输入解锁码等重复了，将他们调整到同一个地方
5、使用新的快照版本来进行更新避免上面3、4点出现的问题

**Comment by Bad0RANG3** [comment]
更新快照版的过程最好如下：
下载快照版（old）→复制识别码→爱发电验证识别码→爱发电返回解锁码→输入验证后直接更新并且去掉提示
没有23.33的话直接去识别启动器更新就好了
希望尽快解决，电脑换来换去的对于我来说也确实不方便qwq

**Comment by Chiloven945** [comment]
资源引导帮助页面有

**Comment by zkitefly** [comment]
提个issue吧，要记得找找有没有重复的issue！

~~但感觉这些玩意都不会搞的（~~

**Comment by WForst-Breeze** [comment, marked as answer]
1. 你可以用自定义主页来实现该内容，个人感觉这个功能也不是启动器的必要功能，可以交个 issue，不过通过的概率不是很大就是了qwq
2. 可以去新功能投票中投票
4. #2130
